### DISCUSSION TITLE
I encountered a build error while doing the GitHub Packages skill

### DISCUSSION THREAD

**Discussion by rfalanga** [skills/.github#128] [Q&A] [answered]
In Step 3 of the GitHub Packages skill, when I completed the PR to merge the `ci` branch after having added the publish.yml and Dockerfile files, I received an error from the build:

> [publish](https://github.com/rfalanga/intro-to-github-packages/actions/runs/3543332366/jobs/5949687751)
> Node.js 12 actions are deprecated. For more information see: https://github.blog/changelog/2022-09-22-github-actions-all-actions-will-begin-running-on-node16-instead-of-node12/. Please update the following actions to use Node.js 16: actions/checkout@v2
> 
> [publish](https://github.com/rfalanga/intro-to-github-packages/actions/runs/3543332366/jobs/5949687751#step:4:1)
> Input 'repository' has been deprecated with message: v2 is now available through docker/build-push-action@v2

Where do I make the necessary changes, so that the GitHub Packages training module will work?

**Comment by rtreddick** [comment]
I got the same error during the first step of hello-github-actions. I first noticed something was wrong because the README said it would automatically close the current step and open the next one once the action ran, but it failed to do so. I also don't know what the fix is.

**Reply by sinsukehlab** [reply]
That is another error.
See: skills/action-update-step#6

**Comment by sinsukehlab** [comment, marked as answer]
Those actions indicated should be bumped to a later version.
- actions/checkout@v2 to v3
Resolved in: skills/publish-packages@2459f17

- docker/build-push-action@v1 to v2 or later (v4 is the latest)

**Reply by sinsukehlab** [reply]
Related issue: skills/publish-packages#17
Related PR: skills/publish-packages#19

**Reply by sinsukehlab** [reply]
The PR has been merged.

**Reply by sinsukehlab** [reply]
Thanks for selecting this as the answer.
